kerlescan/config.py

```diff
@@ -13,12 +14,37 @@ def str_to_bool(s):
     raise ValueError("Unable to determine boolean value from given string argument")
 
 
+def load_hosts_setting(env_name, clowder_endpoint, default):
+    if isClowderEnabled():
+        cfg = LoadedConfig
+
+        final_endpoint = ""
+        for endpoint in cfg.endpoints:
+            if endpoint.app == clowder_endpoint:
+                final_endpoint = f"http://{endpoint.hostname}:{endpoint.port}"
+                return final_endpoint
+
+        if final_endpoint == "":
+            return default
+    else:
+        return os.getenv(env_name, default)
+
+
+inventory_svc_hostname = load_hosts_setting(
+    "INVENTORY_SVC_URL", "host-inventory", "http://inventory_svc_url_is_not_set"
+)
+
+baseline_svc_hostname = load_hosts_setting(
+    "BASELINE_SVC_URL", "baseline", "http://baseline_svc_url_is_not_set"
+)
+
+rbac_svc_hostname = load_hosts_setting("RBAC_SVC_URL", "rbac", "http://rbac_svc_url_is_not_set")
+
+hsp_svc_hostname = load_hosts_setting("HSP_SVC_URL", "hsp", "http://hsp_svc_url_is_not_set")
+
+drift_svc_hostname = load_hosts_setting("DRIFT_SVC_URL", "drift", "http://drift_svc_url_is_not_set")
```

Comment:
Will be great if we move forward with constants in uppercase: https://www.python.org/dev/peps/pep-0008/#constants
```suggestion
INVENTORY_SVC_HOSTNAME = load_hosts_setting(
    "INVENTORY_SVC_URL", "host-inventory", "http://inventory_svc_url_is_not_set"
)

BASELINE_SVC_HOSTNAME = load_hosts_setting(
    "BASELINE_SVC_URL", "baseline", "http://baseline_svc_url_is_not_set"
)

RBAC_SVC_HOSTNAME = load_hosts_setting("RBAC_SVC_URL", "rbac", "http://rbac_svc_url_is_not_set")

HSP_SVC_HOSTNAME = load_hosts_setting("HSP_SVC_URL", "hsp", "http://hsp_svc_url_is_not_set")

DRIFT_SVC_HOSTNAME = load_hosts_setting("DRIFT_SVC_URL", "drift", "http://drift_svc_url_is_not_set")
```

Reply:
I don't know the reason of github duplicate my requested changes. Resolving this.